process_manual/changing_the_process.md

```diff
@@ -0,0 +1,177 @@
+
+## Development process
+This section will describe different thoughts about the process relating to the development.
+Generally we tried to choose as popular technologies as possible, or technologies that seem to have emerging popularity.
+This increases the likelihood that current and future students are familiar with the technologies.
+
+### Language
+We have chosen the Flutter language.
+We changed from Xamarin to Flutter.
+There is a multitude of reasons for choosing flutter.
+
+**It is cross-platform**  
+When you develop something in flutter, it can run on both Android and iOS.
+Some of the users have iPads, and others have Android tablets.
+Instead of developing two apps that look exactly the same, you write the code in one language (Flutter), and run it on both platforms.
+
+**It allows cross-compilation**  
+Xamarin is owned by Microsoft.
+Microsoft does not support Linux products.
+Alas, Xamarin cannot officially be compiled on Linux.
+
+Compiling Xamarin on Linux was doable, but it took a lot of effort to setup, and a lot of effort to actually compile the code.
+Even when finding an efficient way of doing the compilation, it will depend on unofficial ways of doing it.
+
+We had 11 developers running Linux.
+We don't know how many you have, and you don't know how many they have next year.
+Please keep this in mind, if you consider dismissing this point because you don't have a lot of Linux users.
+
+**It compiles to native code**  
+Flutter will compile directly to native iOS and Android code.
+Xamarin runs in a container, meaning it is less efficient, as it has to be translated afterwards.
+
+As Flutter directly compiles to native code it will run much more smoothly and much more efficiently.
+
+**Hot reload**  
+You will love hot reload.
+If you run the app on your phone while developing, every time you hit save, the app will be reloaded in a quarter of a second.
+And it feels that fast.
+
+**Bias of experience**  
+Previous years mentioned changing to another language because "that's what the developers felt most comfortable with."
+No one from this year had any previous experience with Flutter.
+We have no guarantee that you know anything about Flutter, and you don't know if next year will.
+
+As such, this argument should be dismissed entirely.
+Considering experience is only relevant in a very short term.
+Instead, accessibility and documentation has long term relevancy, as this is what will help all the developers that don't know the language.
+
+A few developers with a lot of experience can't lift the entire project anyway.
+And they definitely can't help next year.
+
+### GitHub
+We changed from GitLab to GitHub.
+GitLab was hosted on one of AAU's servers, but we choose to let GitHub host the project.
+
+We did this for two reasons:
+
+**Self-hosting gives you the responsibility**  
+If the GitLab solution was kept, where we had to self-host, we, as students, would be responsible for server maintenance on this part.
```

Comment:
```suggestion
If the GitLab solution which we had to self-host was kept, we would be responsible for server maintenance on this part.
```
Normally, I don't comment for commas, but this was an exceptional case. What is "this part"?